eng/pipelines/corefx-base.yml

```diff
@@ -196,7 +196,6 @@ jobs:
                 msbuildScript: $(_msbuildCommand)
                 framework: $(_framework)
                 outerloop: $(_outerloop)
-                enableAzurePipelinesReporter: false
```

Comment:
This change is not enough. The parameter has a default value in `helix.yml` to be `''` and passed down as a global property to `sendtohelix.proj`. We need to remove that parameter completely from `helix.yml`

https://github.com/dotnet/corefx/blob/master/eng/pipelines/helix.yml#L32

Reply:
That is why the build is failing 😄